src/python-common/ceph/deployment/service_spec.py

```diff
@@ -3465,12 +3467,7 @@ def _derive_cluster_uri(self, uri: str, objname: str) -> str:
         return uri
 
     def _default_ports(self) -> Dict[str, int]:
-        return {
-            'smb': 445,
-            'smbmetrics': 9922,
-            'ctdb': 4379,
-            'remote-control': 54445,
-        }
+        return dict(smbconst.DEFAULT_PORTS)
```

Comment:
`smbconst.DEFAULT_PORTS` is already a _dict_.

Reply:
This is intentional because I don't want a caller mutating the return from the function causing the changes to the constant. I did a little reasearch and found types.MappingProxyType, but it seems a bit overkill for my needs. Regardless, this is staying for now.